### ISSUE TITLE
Auto path suggestion does not work after `bibliography: ` metadata field in YAML

### ISSUE THREAD

**Issue by jooyoungseo** [quarto-dev/quarto-vscode#86]
I was wondering if auto path suggestion could be supported in `bibliography: ` metadata field in YAML. It does not seem to work at this point.

**Comment by jjallaire** [comment]
that's correct, none of our YAML completion currently supports paths (but it's definitely on our list to do this!)
